index.css

```diff
@@ -0,0 +1,56 @@
+@tailwind base;
+@tailwind utilities;
+@tailwind components;
+
+@font-face {
+  font-family: "primary";
+  src: url("./assets/font/primary-font.woff2");
+}
+
+@font-face {
+  font-family: "secondary";
+  src: url("./assets/font/secondary-font.woff2");
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+#gradient-canvas {
+  --gradient-color-1: #ffcb57;
+  --gradient-color-2: #90e0ff;
+  --gradient-color-3: #a960ee;
+  --gradient-color-4: #ff333d;
+}
```

Comment:
in  index.css we supposed not to put there anything..  style it in index.html if you want any  customization put  it in tailwind config